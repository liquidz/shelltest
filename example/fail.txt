bash$ (exit 1); echo $?
0
bash$ echo foo
foo
