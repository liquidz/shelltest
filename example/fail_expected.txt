$ shelltest example/fail.txt

0) (exit 1); echo $?
   command : (exit 1); echo $?
   expected: [{equals 0}]
   actual  : 1


x.

2 tests, 1 failures
